.changeset/fifty-radios-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/brevo-api": minor
```

Comment:
I would say this is a patch. I differentiate it like this:

patch: bugfixes, security updates or small changes
minor: New Features like adding test list functionality
major: Breaking Changes, big new Features that greatly change the use of the module and so on

Reply:
Thank you for the explanation! :)